### ISSUE TITLE
Homebases status lose sync with HA

### ISSUE THREAD

**Issue by Sparenzi** [fuatakgun/eufy_security#465]
First issue: when Homebase status become ARM HOME, the card is in sync, then changing status of the homebase the card does not change.
![immagine](https://user-images.githubusercontent.com/99603311/191317808-54d611c5-e9b8-4546-a238-2f659e03a4b2.png)

Second issue is that no record of the changes is present in the log 
`2022-09-20 16:39:24.244  INFO  Eufy Security server listening on host 0.0.0.0, port 3000
2022-09-20 16:39:25.798  INFO  Connected to station Txxxxxxxxxxxx on host 192.168.1.79 and port 24757 
2022-09-20 16:39:34.536  INFO  Push notification connection successfully established `

BTW there is a 2 hours difference between my time and the log time stamp, I don't know why.

I'm using:

- Home Assistant 2022.9.5 
- Supervisor 2022.09.1 
- Operating System 9.0 on a NUC
- Frontend 20220907.2 - latest 
- Eufy Integration 0.94
- Eufy Security 2.7.0

I'm not an expert, how can I get more info to understand what is going on?
I've already tried to reinstall all the modules.

**Comment by haggis663** [comment]
A docker image with bropat/eufy-security-ws:1.0.0-rc.1 instead of Eufy Integration 0.94 seems to fix the problem.

**Comment by Sparenzi** [comment]
> A docker image with bropat/eufy-security-ws:1.0.0-rc.1 instead of Eufy Integration 0.94 seems to fix the problem.

I have tried your method and the card in lovelace is correctly updated.

I'm not an expert, I have unistalled the 0.9.4 add on and the new one using the docker command.
`docker run -it -e USERNAME=email@address.com -e PASSWORD=password_goes_here -e COUNTRY=country_code -p 3000:3000 bropat/eufy-security-ws:1.0.0-rc.1`
How to make it stick after a reboot?
